gulp/build.js

```diff
@@ -96,8 +101,28 @@ module.exports = {
 
 		const writeFilesPromise = Promise.all(_.map(allExports, (exportName) => {
 			return new Promise((resolve, reject) => {
-				const exportCode = `export * from '${importMap[exportName]}';`;
-				// for some reason this doesn't pickup our .babelrc even though it's supposed to
+				const specifierPath = importMap[exportName].path;
+				const specifierType = importMap[exportName].type;
+
+				// Sorry, this is a little hard to read, it was hard for me to figure
+				// out :facepalm:
+				const exportCode = specifierType === 'ImportSpecifier' ? `
```

Comment:
There are three kinds of imports, this covers all three of those cases and most importantly ensures there's a default export in each case.